ConsoleGame/ConsoleGame/Character.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace ConsoleGame
+{
+    public class Character
+    {
+        public int XPosition { get; private set; }
+        public int YPosition { get; private set; }
+
+        public Character(string gameField)
+        {
+            var currentCharactertPosition = GetStartCharacterPosition(gameField);
```

Comment:
```suggestion
            (int X, int Y) currentCharactertPosition = GetStartCharacterPosition(gameField);
```
Тогда вместо `Item1` и `Item2` будут адекватные имена